crud/registry.go

```diff
@@ -117,7 +117,7 @@ func (r *Registry) Do(kind Kind, op Op, args ...Arg) (Arg, error) {
 	case Delete.name:
 		res, err = a.Delete(args...)
 	default:
-		return nil, errors.New("unknown operation: " + op.name)
+		return nil, fmt.Errorf("unknown operation: " + op.name)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unknown operation: %s", op.name)
```

Again - don't rely on inputs when constructing the format string.